config.ru

```diff
@@ -4,4 +4,10 @@ $LOAD_PATH.unshift("#{File.dirname(__FILE__)}/lib")
 require 'lib/env'
 require 'lib/mediators'
 
-run API::App
+map '/' do
```

Comment:
For consistency, maybe map this to `/api` and remove the `prefix :api` from `lib/api/app.rb`